scene/2d/path_2d.cpp

```diff
@@ -227,6 +232,15 @@ void PathFollow2D::_update_transform() {
 
 void PathFollow2D::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_READY: {
+			if (Engine::get_singleton()->is_editor_hint()) {
+				update_timer = memnew(Timer);
+				update_timer->set_wait_time(0.5);
```

Comment:
missing the 'f' suffix off the "0.5"; even tho the compiler will read it as a float anyway due to the param type

Reply:
f suffix isn't widely used in the codebase, so it's ok.